packages/flutter/lib/src/cassowary/constraint.dart

```diff
@@ -5,21 +5,46 @@
 import 'priority.dart';
 import 'expression.dart';
 
+/// A constraint is created by specifying a relationship between two
```

Comment:
Add something like the following before this:
```
/// Relationships between expressions.
///
```

The idea here is to have a shortish sentence fragment at the top of each description which can be slotted into the following sentence:

"The `Foo` type is ..."

That way when you're scanning the docs on docs.flutter.io you can get a quick reminder of what each thing is, and if you need more you can click on it to get the full docs.